tools/pd-simulator/main.go

```diff
@@ -46,12 +46,15 @@ var (
 	caseName       = flag.String("case", "", "case name")
 	serverLogLevel = flag.String("serverLog", "fatal", "pd server log level.")
 	simLogLevel    = flag.String("simLog", "fatal", "simulator log level.")
+	regionNum      = flag.Int("regionNum", 10000, "regionNum")
+	storeNum       = flag.Int("storeNum", 6, "storeNum")
 )
 
 func main() {
 	flag.Parse()
 
 	simutil.InitLogger(*simLogLevel)
+	simutil.TransferRegionCounter.Init(*storeNum, *regionNum)
```

Comment:
How about the other cases?

Reply:
It only init the counter.

Reply:
It has update to whether loading according the flags.